packages/lodestar/src/chain/blocks/verifyBlock.ts

```diff
@@ -272,6 +279,15 @@ export async function verifyBlockStateTransition(
           errorMessage: execResult.validationError,
         });
     }
+
+    if (isMergeTransitionBlock) {
```

Comment:
Add nice comment here like in Lighthouse explaining why we do this here and not in the forkchoice. Like also to the spec's location where this code should be